.github/workflows/release.yml

```
@@ -1,74 +1,74 @@
-        uses: peter-evans/repository-dispatch@v2
-        with:
-          token: ${{ secrets.TAP_REPO_PAT }}
-          repository: mostlygeek/homebrew-llama-swap
-          event-type: new-release
-          client-payload: |
-            {
-              "release": {
-                "tag_name": "${{ steps.tag.outputs.tag }}"
-              }
-            }
+name: goreleaser
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)